src/main.ts

```diff
@@ -54,6 +70,14 @@ async function main() {
 		}`
 	);
 
+	const isPublicUrl: string = publicWsUrls.includes(config.SUBSTRATE.WS_URL)
+		? 'PUBLIC'
+		: 'PRIVATE';
+
+	logger.info(
+		`[${isPublicUrl} URL] ${config.SUBSTRATE.WS_URL} is a ${isPublicUrl} URL`
+	);
+
```

Comment:
@joepetrowski is there any specific message you have in mind to encourage production level users to setup their own infrastructure?

Reply:
Don't log anything if it's private. If it's public, "`${config.SUBSTRATE.WS_URL} is a public node. Too many users will overload this public endpoint. Switch to a privately hosted node when possible."